.github/workflows/examples-tfplan-tests.yml

```diff
@@ -18,24 +34,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        tf_working_dir:
-          [
-            examples/analytics/emr-on-eks,
-            examples/analytics/spark-k8s-operator,
-            examples/gitops/argocd,
-            examples/aws-efs-csi-driver,
-            examples/crossplane,
-            examples/eks-cluster-with-new-vpc,
-            examples/fully-private-eks-cluster,
-            examples/game-tech/agones-game-controller,
-            examples/ingress-controllers/nginx,
-            examples/karpenter,
-            examples/multi-tenancy-with-teams,
-            examples/node-groups/managed-node-groups,
-            examples/node-groups/self-managed-node-groups,
-            examples/node-groups/windows-node-groups,
-            examples/node-groups/fargate-profiles
-          ]
+        tf_working_dir: ${{ fromJson(needs.getExampleDirectories.outputs.directories) }}
```

Comment:
validate #allTheThings 

For the ones we know won't pass (i.e. - external DNS where we don't have a proper domain to use yet), we can add a filter/exclusion for just those examples